student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < person
```

Comment:
* Kindly write your `person ` class in `Upper case ` as you will get` unknown method or variable person` HINT 💭 you can write this line as `class Student < Person ` 👍